objectives/results/_add_sub_stat.py

```diff
@@ -5,16 +5,13 @@
 from objectives.results._apply_characters_party import ApplyToCharacter, ApplyToCharacters
 
 def AddStat(stat_address):
+    MAX_STAT = 0x80
     return [
         asm.LDA(stat_address, asm.ABS_Y),
-        asm.CLC(),
         asm.ADC(field.LongCall.ARG_ADDRESS, asm.DIR),
-        asm.BCS("MAXIMUM"),
-        asm.BRA("STORE"),
-
-        "MAXIMUM",
-        asm.LDA(255, asm.IMM8),
-
+        asm.CMP(MAX_STAT, asm.IMM8),
+        asm.BLT("STORE"),             # If < 128, skip to STORE
+        asm.LDA(MAX_STAT, asm.IMM8),  # Else, cap at 128
```

Comment:
I think just the 255 value should be changed (and adding a named variable for the value is a good idea).  While not currently possible, this does not handle overflow and could be a hard bug to track down if it ever occurred (e.g. if someone raised the max objective stat result to 128). Performance wise, with no overflow it looks like they take the same amount of cycles/space (bcs which won't happen + bra compared to cmp + blt that will happen) so the safety seems free (although I am usually willing to trade cycles/space for safety anyways).

Reply:
I may be misunderstanding, but do you mean that I could have just changed this line? `asm.LDA(255, asm.IMM8),` with a 128? 

I had tried that and it did not have the desired effect (I still observed stats above 128) because the carry bit wasn't getting added unless the addition put it above 255. 

(Sorry if I'm misunderstanding)

Reply:
Oh, thank you! Sorry, I got changes mixed up while testing things. This looks good to me.